### ISSUE TITLE
format not given for circos plot in other species

### ISSUE THREAD

**Issue by Yogesh1-11** [123xiaochen/VCFshiny#10]
i can not find description on the data required to construct backbone of circos 
![image](https://github.com/user-attachments/assets/43b8e8bc-2dee-41ac-b2be-10d4e883512f)

**Comment by 123xiaochen** [comment]
Thank you for your question. We have provided as many species as possible for drawing. Please look forward to it.

**Comment by Yogesh1-11** [comment]
Hi I am into plant sciences and i couldnt find any plant species in there. Is it possible to upload chromosome start end points directly?

**Comment by 123xiaochen** [comment]
Yes, if you don't have the species selection you need in VCFshiny, you can build a txt file similar to a chart to upload for analysis
![image](https://github.com/user-attachments/assets/5eaf9fae-468b-40dd-8e23-b7984ba27577)

**Comment by Yogesh1-11** [comment]
Thank you chen
could you please provide a demo format text to upload. in case anyone needs
it in future.

**Comment by 123xiaochen** [comment]
I uploaded a sample file in the inst/extdata folder, which matches the built-in data.
